# Automatically Generated in 2017-06-12 22:33:47.452899
# MESSAGE: ODOMETER_RAW
# Description:Measurement of Odometry Sensor (raw ticks)
uint8 ID = 137
uint8 sysid
uint8 compid
uint32 timestamp
int32 xdist
int32 ydist
int32 zdist
int16 quality
